spacy/pipeline/entity_linker.py

```diff
@@ -540,7 +540,7 @@ def predict(self, docs: Iterable[Doc]) -> ActivationsT:
                                 doc_scores=doc_scores,
                                 doc_ents=doc_ents,
                                 scores=[1.0],
-                                ents=[candidates[0].entity_],
+                                ents=[candidates[0].entity],
```

Comment:
This type of fix could have been a separate PR, especially considering that it's a bug fix that we could get into `master` without worrying about whether the rest of this PR is to `master` or `v4`

Reply:
I think that doesn't apply here, as `._add_activations()` seems to be a v4 addition. `master` uses [this](https://github.com/explosion/spaCy/blob/master/spacy/pipeline/entity_linker.py#L499), which is correct.

I can split off another v4-targeted PR for this, if you want

Reply:
It's just that `mypy` raises an error due to `str`/`int` type mismatch here (with good reason, I don't know how this passed when it was merged originally), so in case of a PR I'd have to slap on an `type: ignore` in this line.

Reply:
Oh, right. Ok, let's just go ahead and merge it as part of this PR then.